static/src/javascripts/projects/common/modules/commercial/dfp/get-adverts.js

```diff
@@ -0,0 +1,18 @@
+define([
+    'common/modules/commercial/dfp/private/dfp-env',
+    'common/modules/commercial/dfp/private/get-advert-by-id',
+    'common/modules/commercial/dfp/dfp-obj'
+], function (dfpEnv, getAdvertById, dfp) {
+    dfp.getAdverts = getAdverts;
+    return getAdverts;
+
+    function getAdverts(isWithAllAds) {
```

Comment:
What does `isWithAllAds` actually refer to here? What does it mean for an ad to be "with all ads"?